src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/DataExtractionTransformer.java

```diff
@@ -41,8 +41,9 @@
   public DataExtractionTransformer(boolean isContributionsLatestEndpoint,
       boolean isContributionsEndpoint, ExecutionUtils exeUtils, boolean clipGeometries,
       String startTimestamp, InputProcessingUtils utils, Set<SimpleFeatureType> simpleFeatureTypes,
-      FilterExpression filter, Set<Integer> keysInt, boolean includeTags, boolean includeOSMMetadata,
-      boolean includeContributionTypes, ElementsGeometry elementsGeometry, String endTimestamp,
+      FilterExpression filter, Set<Integer> keysInt, boolean includeTags,
+      boolean includeOSMMetadata, boolean includeContributionTypes,
```

Comment:
Why is includeOSMMetadata  now in the line 45? it fits in the line 44 how it was before?

Reply:
AFAICS the line would have been over 100 characters long if `boolean includeOSMMetadata,` were left on line 44: https://jenkins.ohsome.org/job/ohsome-api/job/master/509/checkstyle/fileName.-1056595858/source.f9714832-4e01-442a-a293-54fef9cd83e7/#44

Reply:
Ok eclipse shows the line-width marker at character 101 even if it is set at 100. That is why I didn't see it.